Userland/Applications/PixelPaint/Tools/BrushTool.cpp

```diff
@@ -185,9 +185,34 @@ GUI::Widget* BrushTool::get_properties_widget()
 
 NonnullRefPtr<Gfx::Bitmap> BrushTool::build_cursor()
 {
-    m_scale_last_created_cursor = m_editor ? m_editor->scale() : 1;
-    auto scaled_size = size() * m_scale_last_created_cursor;
+    float new_scale = m_editor ? m_editor->scale() : 1;
```

Comment:
Nit: `float` -> `auto`

Reply:
Why use auto over float is there a benefit for this?
I'll make the change though :)

Reply:
The serenity style is pretty much always use `auto`